src/index.ts

```diff
@@ -50,7 +50,7 @@ ethers.utils.Logger.setLogLevel(ethers.utils.Logger.levels.ERROR);
  * @returns {object} Returns an instance of the Compound.js SDK.
  */
 const Compound = function(
-  provider: Provider | string = 'mainnet', options: CompoundOptions = {}
+  provider: Provider | string = 'rsk-testnet', options: CompoundOptions = {}
```

Comment:
```suggestion
  provider: Provider | string = 'testnet', options: CompoundOptions = {}
```
Use same name as in constants